src/component/App.js

```diff
@@ -1,14 +1,39 @@
-import '../App.css';
+/* eslint-disable react/sort-comp */
+import './App.css';
+import React from 'react';
+import calculate from '../logic/calculate';
 import Display from './Display';
 import ButtonShape from './ButtonShape';
 
-function App() {
-  return (
-    <>
-      <Display />
-      <ButtonShape />
-    </>
-  );
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  get result() {
+    const { next, total, operation } = this.state;
+
+    return `${next || ''}${total || ''}${operation || ''}`;
+  }
+
```

Comment:
- [ ] The way this prop is coded and passed to the Display component is producing an awkward output. Please see below. What I tried to do was add two numbers together.
![Screenshot 2021-08-02 at 12-37-05 React App](https://user-images.githubusercontent.com/70150423/127863024-217f5faa-bad0-47f5-a531-5504fea032a0.png)
You have something like the image above for all your calculator operations. Buttons are concatenated with the operations.